exercises/src/test/scala/pl/japila/scalania/s99/P21Spec.scala

```diff
@@ -2,11 +2,13 @@ package pl.japila.scalania.s99
 
 import org.specs2.mutable._
 import pl.japila.scalania._
+//Fix the import standard
+import S99_P21._
 
 object P21Spec extends Specification with ExamplesBlock {
   "P21 solution" should {
     "Insert an element at a given position into a list." in {
-      import S99_P21._
+
```

Comment:
Revert the change. I'm in favour of having imports as close to the place they are used in as possible.